### ISSUE TITLE
"NewRelicCapacitorPlugin" plugin is not implemented on ios

### ISSUE THREAD

**Issue by Toxiapo** [newrelic/newrelic-capacitor-plugin#53]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

I followed the instrumentation guide for my Ionic Capacitor app. Android is working but I am getting errors on iOS.

https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor/

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT give us access or passwords to your New Relic account or API keys! )

## Steps to Reproduce
- Install agent with [guided install](https://onenr.io/0VRVoxJW1ja)
- build the hybrid app
- run the app on an ios emulator

[NOTE]: # ( Please be as specific as possible. )

## Expected Behavior
I should not see errors on the Safari developer console, and the new relic dashboard should display an App Launch data point

[NOTE]: # ( Tell us what you expected to happen. )

## Relevant Logs / Console output
![Screenshot 2023-09-15 at 3 47 28 PM](https://github.com/newrelic/newrelic-capacitor-plugin/assets/14434233/1ffe4ce8-4890-4dd7-8804-bf5801b05e54)

[NOTE]: # ( Please provide specifics of the local error logs, Browser Dev Tools console, etc. if appropriate and possible. )

## Your Environment

[TIP]:  # ( Include as many relevant details about your environment as possible. )

#### MacBook Pro Intel

- XCode: Version 14.1
- @newrelic/newrelic-capacitor-plugin@1.3.1
- Capacitor@5.0.6

## Additional context

[TIP]:  # ( Add any other context about the problem here. )

I am not super familiar with iOS development,  so there may be set-ups I need to do on xcode? However, when I was following through the guided installation UI, the only instruction needed on the iOS side was to run the command `pod install`, and that's what I did.  While Android did provide[ additional guidelines](https://docs.newrelic.com/docs/mobile-monitoring/new-relic-mobile-ionic-capacitor/get-started/introduction-new-relic-ionic-capacitor/#android-only-if-youre-monitoring-an-android-native-app-follow-these-steps) with working with the android code.

**Comment by ndesai-newrelic** [comment]
@Toxiapo we will review this issue from our side on Monday.

**Comment by Toxiapo** [comment]
Thank you @ndesai-newrelic! Please let me know if you need more context from my side.

(This is a monorepo project with NPM+React framework using node@16.18 and NPM@9.5.1)

**Comment by ndesai-newrelic** [comment]
@Toxiapo can you share example app where you reproduced this issue? I am unable to do it with capacitor react app?

**Comment by Toxiapo** [comment]
@ndesai-newrelic I do not have an example app, but I can try to create an MVP of my app

**Comment by ndesai-newrelic** [comment]
@Toxiapo can you do that? it will be very helpful to us

**Comment by Toxiapo** [comment]
Hi @ndesai-newrelic, I could not replicate the bug using an example app but I believe I know why it was happening. We can close this issue. Thank you!

For anyone who's getting similar errors: The problem was that when I installed the plugin and sync the assets to the ios/android folder, it failed to include the new plugin in the Podfile like the example [shown here](https://github.com/newrelic/newrelic-capacitor-plugin/blob/74c06812ad8551cb35166b9d30ece32db1694d5e/example/ios/App/Podfile#L18). To resolve this, I manually added the missing plugin to the Podfile and everything seems to be working fine.

@ndesai-newrelic  I had assumed that 'npx cap sync' would automatically incorporate new plugins into the native platforms, but I haven't yet determined why it didn't work in my configuration.
